deploy/base/operator.yaml

```diff
@@ -11,6 +11,7 @@ spec:
   template:
     metadata:
       annotations:
+        openshift.io/scc: privileged
```

Comment:
Is it common to have distro-specific configuration like this in general-purpose k8s operators?